test/dom/dom.ts

```diff
@@ -546,9 +546,9 @@ describe("DOM", () => {
 
     // elements returns values for given Struct where for a fieldname it only contains the last value
     // instead of an array of all values
-    assert.equal(41, s.elements()[0]);
+    assert.equal(55, s.elements()[0]);
     assert.isFalse(Array.isArray(s.elements()[0]));
-    assert.equal("Jessie", s1.elements()[0]["first"]);
+    assert.equal("John", s1.elements()[0]["first"]);
```

Comment:
This does exercise the new behavior, but it does so in a pretty subtle way; it returns the first value, which previously was skipped in favor of the second field with the same name.

Could you add some assertions for the other values that we expect `elements()` to return?